crates/store/re_log_encoding/src/encoder.rs

```diff
@@ -120,15 +179,20 @@ impl<W: std::io::Write> Encoder<W> {
         }
     }
 
+    // NOTE: This cannot be done in a `Drop` implementation because of `Self::into_inner` which
+    // does a partial move.
+    #[inline]
     pub fn finish(&mut self) -> Result<(), EncodeError> {
```

Comment:
This should maybe take `self` instead?

Reply:
Can try -- if it doesn't break the world then sure why not

Reply:
Ah no cannot do that otherwise I couldn't implement `finish()` in `DroppableEncoder`, and we need it if people want to do err checking.